src/main/resources/jenkins/branch/NoTriggerBranchProperty/config.jelly

```diff
@@ -24,4 +24,8 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core"/>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+  <f:entry title="${%Suppression strategy}" field="strategyId">
+    <f:select default="1"/>
```

Comment:
I think it should be changed to
```suggestion
    <f:select default="all"/>
```

I'll verify it tomorrow. If I am right, I'll update the PR.

Reply:
Yes, it was a mistake and I fixed it 🙂